SpywareFilter/sections/whitelist.txt

```diff
@@ -990,10 +990,10 @@ nnmclub.ro,nnm-club.t0r.xyz,nnm-club-me.appspot.com,nnm-club.lib,nnm-club.me,nnm
 @@||google-analytics.com/gtm/js$domain=jump.com.tr
 !
 ! Fixing broken web sites
-ciokorea.com,adpiamall.com,live.joins.com,naver.com,jbe.go.kr#%#//scriptlet('set-constant', 'window.wcs_add', 'noopFunc')
-ciokorea.com,adpiamall.com,live.joins.com,naver.com,jbe.go.kr#%#//scriptlet('set-constant', 'window.wcs_do', 'noopFunc')
+privatenote.co.kr,ciokorea.com,adpiamall.com,live.joins.com,naver.com,jbe.go.kr#%#//scriptlet('set-constant', 'window.wcs_add', 'noopFunc')
+privatenote.co.kr,ciokorea.com,adpiamall.com,live.joins.com,naver.com,jbe.go.kr#%#//scriptlet('set-constant', 'window.wcs_do', 'noopFunc')
```

Comment:
Have you checked these scriptlets? I still need the exception rule.

Reply:
Yes. I tested on Mozilla Firefox with AdGuard Browser Extension.
An unblocking rule can be founded at line 996.

Reply:
I meant that on my end those scriplets don't work. It's better to just do an exception rule.

Reply:
Okay.

Reply:
@zloyden

Reply:
I see, but it looks like they don't work.
1. Remove these scriptlets.
2. Leave the exceptional rule without hint.
```
@@||wcs.naver.net/wcslog.js$domain=ciokorea.com|adpiamall.com|live.joins.com|naver.com|jbe.go.kr|privatenote.co.kr
```

Reply:
I've done it here.

Reply:
I cannot understand your meaning.
I think the website works correctly.

<details>
<summary>Firefox with AdGuard Browser Extension</summary>

![Screenshot0](https://user-images.githubusercontent.com/98787049/155094941-95369390-5025-4d3f-92c9-e027c6f23ecb.png)
![Screenshot2](https://user-images.githubusercontent.com/98787049/155094959-99883b27-d34b-45b2-8bcc-a45e6e268361.png)
![Screenshot3](https://user-images.githubusercontent.com/98787049/155094967-f55c3093-ce1e-4332-b4e3-47e01609a07f.png)

</details>

Reply:
Just look at my commit https://github.com/AdguardTeam/AdguardFilters/pull/111085/commits/62860b20c1b660fcc24719bab0a40bc360cd2a59 and I hope you will understand it :)